packages/dapp-toolkit/src/modules/wallet-request/encryption/encryption.module.ts

```diff
@@ -61,7 +61,7 @@ export const EncryptionModule = () => {
    *
    * @param {Buffer} data - payload to be encrypted
    * @param {Buffer} encryptionKey - key used for encryption
-   * @param {Buffer} iv - optional initialization vector
+   * @param {Buffer=} iv - optional initialization vector
```

Comment:
typo?

Reply:
it's JSDoc way of marking parameter as optional 💯